libs/hdf-converters/src/veracode-mapper.ts

```diff
@@ -0,0 +1,486 @@
+      business_criticality: {path: 'detailedreport.business_criticality'},
+      generation_date: {path: 'detailedreport.generation_data'},
+      veracode_level: {path: 'detailedreport.veracode_level'},
+      total_flaws: {path: 'detailedreport.total_flaws'},
+      flaws_not_mitigated: {path: 'detailedreport.flaws_not_mitigated'},
+      teams: {path: 'detailedreport.teams'},
+      life_cycle_stage: {path: 'detailedreport.life_cycle_stage'},
+      planned_deployment_date: {path: 'detailedreport.planned_deployment_date'},
+      last_update_time: {path: 'detailedreport.last_update_time'},
+      is_latest_build: {path: 'detailedreport.is_latest_build'},
+      policy_name: {path: 'detailedreport.policy_name'},
+      policy_version: {path: 'detailedreport.policy_version'},
+      policy_compliance_status: {path: 'detailedreport.plolicy_compliance_status'},
+      policy_rules_status: {path: 'detailedreport.plolicy_rules_status'},
+      grace_period_expired: {path: 'detailedreport.grace_period_expired'},
+      scan_overdue: {path: 'detailedreport.scan_overdue'},
+      business_owner: {path: 'detailedreport.business_owner'},
+      business_unit: {path: 'detailedreport.business_unit'},
+      tags: {path: 'detailedreport.tags'},
+      legacy_scan_engine: {path: 'detailedreport.legacy_scan_engtine'},
+      custom_fields: {path: 'detailedreport.custom_fields'}
+
+    },
+    version: HeimdallToolsVersion,
+    statistics: {
+    },
+    profiles: [
+      {
+        name: {path: 'detailedreport.policy_name'},
+        version: {path: 'detailedreport.version'},
+        title: {path: 'detailedreport.static-analysis.modules.module.name'},
+        copyright: 'Copyright Veracode, Inc., 2014.',
+        supports: [],
+        attributes: [],
+        groups: [],
+        status: 'loaded',
+        controls: [
+          {
+            path: 'detailedreport.severity[0].category',
+            ...controlMappingCwe(5)
+          },
+          {
+            path: 'detailedreport.severity[1].category',
+            ...controlMappingCwe(4)
+          },
+          {
+            path: 'detailedreport.severity[2].category',
+            ...controlMappingCwe(3)
+          },
+          {
+            path: 'detailedreport.severity[3].category',
+            ...controlMappingCwe(2)
+          },
+          {
+            path: 'detailedreport.severity[4].category',
+            ...controlMappingCwe(1)
+          },
+          {
+            path: 'detailedreport.severity[5].category',
+            ...controlMappingCwe(0)
+          },
+          {
+            path: 'detailedreport.software_composition_analysis.vulnerable_components',
+            pathTransform: componenttransform,
+            id: {path: "cve_id"},
+            title: {path: 'cve_id'},
+            desc: {path: 'cve_summary'},
+            impact: {path: 'impact'},
+            refs: [],
+            tags: {
+              cwe: {path: 'cwe_id'},
+              nist: {path: 'nist'}
```

Comment:
there's gonna be a pr that'll be merged soon that'll add another tag for ccis, so might need to add it if that one gets merged first

https://github.com/mitre/heimdall2/pull/3170

it'll need to be supported on both the cve and the cwe mappers